main/config_nodes.h

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) The Libre Solar Project Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CONFIG_NODES_H_
+#define CONFIG_NODES_H_
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <stdint.h>
+#include "esp_err.h"
+#include "ts_client.h"
+
+
+#define CONFIG_LIBRE_SOLAR_TYPE_ID 9
```

Comment:
Namespace `CONFIG_` is reserved for auto-generated defines from Kconfig. Please remove the prefix.

Reply:
I would move it to the defaults?